src/components/oracles.js/oracles.js

```diff
@@ -0,0 +1,141 @@
+import paginationFactory from "react-bootstrap-table2-paginator";
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import { Col, Row } from "reactstrap";
+import CryptoIcon from "../../components/CryptoIcon/CryptoIcon.js";
+import DateTimeAgo from "../../components/DateTime/DateTimeAgo.js";
+import Loader from "../../components/Loader/Loader.js";
+import LinkTable from "../../components/Table/LinkTable.js";
+import Value from "../../components/Value/Value.js";
+import ValueChange from "../../components/Value/ValueChange.js";
+import { withErrorBoundary } from "../../hoc.js";
+import { useFetch, usePageTitle } from "../../hooks";
+import { parseUTCDateTimestamp } from "../../utils/datetime.js";
+
+function Ilks(props) {
+  usePageTitle("Oracles");
+  let navigate = useNavigate();
+  const { data, isLoading, isError, ErrorFallbackComponent } =
+    useFetch("aave/oracles/");
+
+  if (isLoading) {
+    return <Loader />;
+  } else if (isError) {
+    return <ErrorFallbackComponent />;
+  }
+
+  const onRowClick = (row) => {
+    navigate(`/oracles/${row.symbol}/`);
+  };
+
+  const assetsTable = (
+    <Row>
```

Comment:
Usage of row and col here is wrong. Only Col should be a child of Row, but you have h4 here as a child of Row. I'd argue here that you don't even need to use Row and Col, but rather just have h4 with className="mb-4" and then directly under that you have LinkTable. Should do the same thing with less elements